File: src/C#/Empleado/claseEmpleado.cs
using System;

public class Empleado
{
    public string Nombre;
    public int anioContratacion;
    public double salario;

    public Empleado()
    {
        this.Nombre = "NULL";
        this.anioContratacion = 0;
        this.salario = 0.0;
    }

    public Empleado(string n, int a, double s)
    {
        this.Nombre = n;
        this.anioContratacion = a;
        this.salario = s;
    }

    public void imprimir()
    {
        Console.WriteLine("\nNombre: " + this.Nombre);
        Console.WriteLine("Anio de Contratacion: " + this.anioContratacion);
        Console.WriteLine("Salario: " + this.salario);
    }
}

File: src/Python/Parcial1/EmpleadoA9/claseEmpleado.py
class Empleado:

    def __init__(self, n="NULL", a=0, s=0.0):
        self.nombre = n
        self.anioContratacion = a
        self.salario = s

    @property
    def nombre(self):
        return self.__nombre

    @nombre.setter
    def nombre(self, n):
        self.__nombre = n

    @property
    def anioContratacion(self):
        return self.__anioContratacion

    @anioContratacion.setter
    def anioContratacion(self, a):
        self.__anioContratacion = a

    @property
    def salario(self):
        return self.__salario

    @salario.setter
    def salario(self, s):
        self.__salario = s

    def toString(self):
        return self.__nombre + " " + str(self.__anioContratacion) + " " + str(self.__salario)

    def calcularSalario(self):
        return self.__salario * 12